rebar.lock

```diff
@@ -22,7 +22,7 @@
   0},
  {<<"aesophia">>,
   {git,"https://github.com/aeternity/aesophia.git",
-      {ref,"faa0ef97725f82dedee9e026c88c428494ce9c2c"}},
+      {ref,"386419f11256407c85ae96c6afcd122fe4d4506f"}},
```

Comment:
I see https://github.com/aeternity/aesophia/compare/faa0ef97725f82dedee9e026c88c428494ce9c2c...386419f11256407c85ae96c6afcd122fe4d4506f

Reply:
It refers to same aebytecode 1526ad3bf057e72a1714aea0430b001bd1d576c9 - that is ok.

Reply:
I see [last node release](https://github.com/aeternity/aeternity/blob/v2.4.0/rebar.lock#L25) had faa0ef97725f82dedee9e026c88c428494ce9c2c - ok.

Reply:
I see that apps/aehttp/priv/bin/aesophia is not touched by this PR (I am trying to get my head around what is impact of this).

Reply:
That is intended.